### PR TITLE
Debug tools

### PR DESCRIPTION
**Pull request by kevingrillet**
Was looking to edit my save for test purposes, then did this.

Correspond to issue #34

Maybe a bit too game breaker...

### CODE REVIEW

File: custom/debugcheats.user.js [hunk 1]
```diff
@@ -0,0 +1,319 @@
+// ==UserScript==
+// @name          [Pokeclicker] Debug Cheats
```
Comment (Ephenia) [inline custom/debugcheats.user.js:2]: The name needs to be following the 3 word naming schema, just like all the other scripts.

File: custom/debugcheats.user.js [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+// @namespace     Pokeclicker Scripts
+// @author        kevingrillet
+// @description   Edit your save for debug (currency, pokeballs, pokemons, ...)
+// @copyright     https://github.com/Ephenia
+// @license       GPL-3.0 License
+// @version       0.6
```
Comment (Ephenia) [inline custom/debugcheats.user.js:8]: The version must start at 1.0.

**Comment by kevingrillet** [comment]
Should be working properly now

**Comment by Ephenia** [comment]
This is approved on votes now.

**Pull request merged**